app/src/components/file-upload/FileUploadItem.tsx

```diff
@@ -203,13 +208,42 @@ const FileUploadItem: React.FC<IFileUploadItemProps> = (props) => {
         <Box display="flex" flexDirection="row" alignItems="center" p={2} width="100%">
           <Icon path={mdiFileOutline} size={1.5} className={error ? classes.errorColor : classes.fileIconColor} />
           <Box pl={1.5} flex="1 1 auto">
-            <Box display="flex" flexDirection="row" flex="1 1 auto" alignItems="center" height="3rem">
+            <Box display="flex" flexDirection="row" flex="1 1 auto" alignItems="center" minHeight="3rem">
```

Comment:
This seems like a case for sx prop (@curtisupshall ?)

Reply:
Yeah I believe all these props can be moved into `sx`. Also, the min height should be absolute, so just use the `sx` units instead of rems.